package.json

```diff
@@ -1,7 +1,7 @@
 {
   "name": "language-shakespeare",
   "version": "0.1.0",
```

Comment:
sure. May I amend the commit (and force push) for cleanliness or would you prefer I make it a new commit?